src/components/modals/DialogModal/DialogModal.tsx

```diff
@@ -0,0 +1,160 @@
+import { CloseIcon } from '../../icons';
+
+function DialogModal({
+  title = '',
+  body,
+  footer,
+  showClose = true,
+  showNext = true,
+  showCancel = true,
+  cancelText = 'Cancel',
+  nextText = 'Next',
+  onClose = () => alert('onClose not set'),
+  onCancel = () => alert('onCancel not set'),
+  onNext = () => alert('onNext not set'),
+}: {
+  title?: string;
+  body?: JSX.Element;
+  footer?: JSX.Element;
+  showClose?: boolean; // shows x and divider for title
+  showNext?: boolean;
+  showCancel?: boolean;
+  cancelText?: string;
+  nextText?: string;
+  onClose?: () => void;
+  onCancel?: () => void;
+  onNext?: () => void;
```

Comment:
these are optional but show alerts when not set?

Reply:
yeah, display and action are set differently, so if display is setup to show a button but the button doesnt have an action it will show the alert that it needs to be configured.